librz/core/agraph.c

```diff
@@ -4027,7 +4027,7 @@ static void graph_breakpoint(RzCore *core) {
 }
 
 static void graph_continue(RzCore *core) {
-	rz_debug_continue_oldhandler(core, "");
+	rz_cmd_debug_continue_execution_handler(core, 1, NULL);
```

Comment:
What about using the proper API instead?
```c
CMD_CHECK_DEBUG_DEAD(core);
rz_cons_break_push(static_debug_stop, core->dbg);
rz_reg_arena_swap(core->dbg->reg, true);
#if __linux__
core->dbg->continue_all_threads = true;
#endif
rz_debug_continue(core->dbg);
dbg_follow_seek_register(core);
```
Or making it some function in the `librz/core/cdebug.c` and using it instead?
Reusing command handlers for pure API purpose doesn't seem right.